clamnibs/beamformer.py

```diff
@@ -61,16 +61,18 @@ def get_target(obj):
         target = np.array([w @ ep for ep in epochs_data]).squeeze()
     else:
         raw_events = mne.events_from_annotations(obj)[0]
-        raw_data = obj.get_data([ixs_goods])
+        raw_data = obj.get_data(ixs_goods)
         epochs = EpochsCLAM(obj)
         epochs_events = epochs.events
-        epochs_data = epochs.get_data([ixs_goods])
+        epochs_data = epochs.get_data(ixs_goods)
         forward_goods = epochs.forward_full[ixs_goods]
         COV = np.mean([np.cov(np.real(np.concatenate(epochs_data[epochs_events[:, 2]
                       == target_code], axis=-1))) for target_code in target_codes], axis=0)
         w = _get_lcmv_weights(COV, forward_goods)
         target = (w @ raw_data).squeeze()
-    target *= obj.flip
+        
+    flip = obj.flip if hasattr(obj, 'flip') else 1
+    target *= flip
```

Comment:
Both RawCLAM and EpochsCLAM objects should always have a flip (see previous comment).

Reply:
The first thing set_flip() does is call get_target(obj), which then tries to use the flip at the end before returning the target, which causes an error because no flip has been set at the time of the get_target call. I can remove handling that case from get_target and move it to set_flip.